<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoordinateConverterDialog</class>
 <widget class="QDialog" name="CoordinateConverterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Converter Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>20</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Converter Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QRadioButton" name="RadioButton_XtionCoordinateConverter">
        <property name="text">
         <string>Xtion Default</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RadioButton_AistCoordinateConverter">
        <property name="text">
         <string>Aist Internal Calibration</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox_ResultButtons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
